src/startup.ts

```diff
@@ -12,15 +12,16 @@ function validateOptions() {
 		rtorrentRpcUrl,
 		qbittorrentUrl,
 		transmissionRpcUrl,
+		delugeWebPassword,
+		delugeWebUrl,
 		dataDirs,
 		linkDir,
 		matchMode,
 		skipRecheck,
 	} = getRuntimeConfig();
 	if (
 		action === "inject" &&
-		!(rtorrentRpcUrl || qbittorrentUrl || transmissionRpcUrl)
-	) {
+		!(rtorrentRpcUrl || qbittorrentUrl || transmissionRpcUrl || (delugeWebUrl && delugeWebPassword))) {
 		throw new CrossSeedError(
 			"You need to specify --rtorrent-rpc-url, --transmission-rpc-url, or --qbittorrent-url when using '-A inject'."
```

Comment:
remember to add --deluge-web-url here later

Reply:
oops, oversight.